db/custom_api/views.py

```diff
@@ -24,3 +25,9 @@ def time_series_api_interface(domain, varname, freq, inner_path=''):
     this_variable = Indicator(domain, varname, freq, inner_path)
     data = DatapointOperations.select(**this_variable.query_param)
     return publish_csv(data)
+
+
+@custom_api_bp.route(f'{BASE_URL}/<string:freq>/info')
+@custom_api_bp.route(f'{BASE_URL}/<string:freq>/<path:inner_path>/info')
+def info(varname, freq, **kwargs):
+    return variable_info(varname, freq)
```

Comment:
Both api.variable_info() and  custom_api.info() must address same utility fucntion that creates a dictionary with variable info. Calling api.variable_info() is not encouraged.